docs/kopia_troubleshooting.md

```diff
@@ -123,4 +114,184 @@ Users may also find the following type of errors in the logs while maintenance i
+* index-compaction - merges multiple smaller index blobs (n) into larger ones
+
+
+#### Full maintenance
+The main purpose of full maintenance is to perform garbage collection of contents that are no longer needed after snapshots they belong to get deleted or age out of the system.
+
+* snapshot-gc - finds all contents (files and directory listings) that are no longer reachable from snapshot manifests and marks them as deleted. It also undeletes contents that are in use and have been marked as deleted before (due to unavoidable race between snapshot gc and snapshot create possible when multiple machines are involved).
+NOTE: This is the most costly operation as it requires scanning all directories in all snapshots that are active in the system. The good news is that all this data is in q blobs and thanks to the quick maintenance it was kept nice and compact and quick to access, so this phase does not usually take that long (e.g. currently ~25 seconds on my 720 GB repository with >1.5M contents).
+
+* full-drop-deleted-content - removes contents that have been marked for deletion long enough from the index. This creates “holes” in pack blobs and/or makes blobs completely unused and subject to deletion.
+
+* full-rewrite-contents - same as quick-rewrite-contents but acts on all blobs (p and q)
+
+* full-delete-blobs - same as quick-delete-blobs but acts on all blobs (p and q)
+
+There are additional safety measures built into the maintenance routine to make it safe to run even when other kopia clients on other machines are executing snapshots concurrently. For example {quick}-delete-blobs will not run if less than X amount of time has passed since last content rewrite and full-drop-deleted-content will only drop contents if enough time has passed between full maintenance cycles.
+
+The recommendation is to run quick maintenance as frequently as it makes sense for your repository (hourly is typically fine). The entire quick cycle should take <10 seconds, even for big repositories.
+
+Full maintenance cycle runs every 24h by default and can be spread apart further (weekly or even monthly is probably fine) or stopped completely if somebody does not want or care to reclaim unused space.
+
+## Kopia Repository Maintenance in OADP
+
+In the namespace where OADP is installed repo-maintain-job's are executed
+
+```shell=
+pod/repo-maintain-job-1730739882527-2nbls                             0/1     Completed   0          168m
+pod/repo-maintain-job-1730743482536-fl9tm                             0/1     Completed   0          108m
+pod/repo-maintain-job-1730747082545-55ggx                             0/1     Completed   0          48m
+pod/repo-maintain-job-1730749183178-5jqf2                             0/1     Completed   0          13m
+pod/repo-maintain-job-1730749483183-mvrzw                             0/1     Completed   0          8m57s
+pod/repo-maintain-job-1730749783183-8vtjh                             0/1     Completed   0          3m57s
+```
```

Comment:
Can this be also referenced to the `kopia maintenance status` in the pod? So reader knows this is the entry point to the information that can be further explored with the commands that were previously described in this doc?

Reply:
ya.. I can do that..